packages/asc-ui/src/components/CompactPager/CompactPager.test.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react'
+import { render, fireEvent } from '@testing-library/react'
+import { ThemeProvider, ascDefaultTheme } from '@datapunt/asc-core'
+import CompactPager, { determineTotalPages } from './CompactPager'
+
+const PREV_BUTTON = 'prevbutton'
+const NEXT_BUTTON = 'nextbutton'
+
+describe('CompactPager', () => {
+  it('should change the page when pressing prev button', () => {
+    const onChangePageMock = jest.fn()
```

Comment:
You're setting the same mock function and mock component four times, defining it just once is enough

Reply:
The arguments passed in are different sometimes, but yeah, I'll add it to a function so it can be re-used.